src/bootstrap-table.js

```diff
@@ -913,7 +913,11 @@ class BootstrapTable {
         return
       }
 
-      if (currentTarget === Utils.getSearchInput(this)[0] || $(currentTarget).hasClass('search-input')) {
+      if (
+        (currentTarget instanceof jQuery && currentTarget.is(Utils.getSearchInput(this))) ||
+        (!(currentTarget instanceof jQuery) && currentTarget === Utils.getSearchInput(this)[0]) ||
+        $(currentTarget).hasClass('search-input')
```

Comment:
At the end we could convert currentTarget to a jQuery object and then evaluate if that object has the corresponding class.. I don't think we could have performance hit if we create a jQuery object from a jQuery object... Do you know that?

Reply:
I'm not sure what do you mean, we don't create a jQuery object on that if.
We just check if the currentTarget is a jQuery object, if it is a jQuery we compare it using the `is` function.
If it is a dom element we get the dom element from the jquery object and just compare it with `===`

Reply:
What about this?
```js
const searchInput = Utils.getSearchInput(this)
currentTarget = $(currentTarget)
if (
        currentTarget.is(searchInput) ||
        currentTarget.hasClass('search-input')
...
```

Reply:
I guess creating a new jquery object will cost more performance.
So i could invert it, i can check if it is a jQuery Object and set `currentTarget` to a domElement.
In the if i would use the vanilla js functions for the comparison.

What do you mean ? :-)